### DISCUSSION TITLE
Can helix be installed on a Linux

### DISCUSSION THREAD

**Discussion by rcortezk9** [helix-editor/helix#5518] [General] [answered]
I do see that there is some Linux distros listed in the installation page but I don't see I don't see all distros. Can it be installed in Pop_os?

**Comment by the-mikedavis** [comment, marked as answer]
There's a new PPA you could use: https://github.com/helix-editor/helix/issues/3640#issuecomment-1373906378 (if I remember correctly, Pop_OS can use PPAs for packages?)

You could also use one of the binaries or the AppImage from the release assets: https://github.com/helix-editor/helix/releases/latest or build from source if you have a rust toolchain already installed
